problems/strings.js

```diff
@@ -120,7 +199,14 @@ function middleCharacter() {}
  * @returns {string}
  */
 
-function reverseString() {}
+// function reverseString() {}
+const reverseString = (string) => {
+  let newString = ""
+  for(i = string.length-1; i>=0; i -= 1) {
+    newString += string.charAt(i)
```

Comment:
we don't want to use charAt we want to use the index the same way we would with arrays. 
Also take notice that we don't want the sentence reversed, we want the spelling of the words to be reversed